src/grid/interpolate.jl

```diff
@@ -98,3 +99,48 @@ linear interpolation of data(x, y)
     gx = (g0 * dy1 + g1 * dy0) / (dx0 + dx1) / (dy0 + dy1)
     return gx
 end
+
+
+"""
+testInterpolation1D(func, grid1, grid2)
+
+Test interpolation on grid1 with function func, compare results with func(grid2).
+Return max deviation and std of deviation.
+
+"""
+function testInterpolation1D(func, grid1, grid2)
+    data1 = []
+    for i in 1:grid1.size
+        push!(data1, func(grid1[i]))
```

Comment:
You may do it with the pythonic way
`data1=[func(grid1[i]) for I in range(grid1.size)]`